style.css

```diff
@@ -0,0 +1,232 @@
+      padding: 5px 0;
+      border-left: none;
+  }
+  .search-btn img{
+      width: 100%;
+  }
+  .user-options{
+  
+  margin-right: auto;
+   margin: 20px;
+  }
+  .side-bar{
+  
+  position: fixed;
+  top: 63px;
+  left: 0;
+  min-width: 250px;
+  width: 10px;
+  height: calc(100vh - 60px);
+  background: #fff;
+  padding: 10px ;
+  }
+  .links{
+      display: block;
+      width: 100%;
+      padding: 10px 20px;
+      display: flex;
+      align-items: center;
+      text-transform: capitalize;
+      font-size: 15px;
+      font-weight: 500;
+      font-family: Verdana, Geneva, Tahoma, sans-serif;
+      color: #242424;
+      text-decoration: none;
+  }
+  links img{
+       height: 25px;
+       margin-right: 20px;
+  
+  }
+  .links:hover, .links.active{
+      background: rgba(0, 0, 0, 0.2);
+  }
+  .separator{
+      border: none;
+      border-bottom: 1px solid rgba(0, 0, 0, 0.2);
+      margin: 20px 0;
+  }
+  .filter{
+      position: fixed;
+      left: 250px;
+      top: 60px;
+      width: calc(100% - 250px);
+      height: 60px;
+      background: whitesmoke;
+      border-top: 1px solid #e4d3d3;
+      border-bottom:1px solid #e2d8d8 ;
+      border-radius: 5px;
+      padding: 0 20px;
+      display: flex;
+      align-items: center;
+  }
+  
+  .filter-option{
+      color: rgb(182, 136, 50);
+      flex: 0 0 auto;
+      padding: 10px 20px;
+      border-radius: 50px;
+      margin-right: 50px;
+      border: 1px solid lightgrey;
+      font-size: 15px;
+      text-transform: capitalize;
+      background: white;
+  
+  
+  }
+  .filter-option:hover, .filter-option.active{
+  background:rgba(0, 0, 0, 1) ;
+  cursor: pointer;
+  }
+
+  .video-container{
+      width: calc(100% - 250px);
+      margin-top: 120px;
+      margin-left:250px ;
+      font: 1em sans-serif;
+      padding: 20px;
+      display: flex;
+      justify-content: end;
+      align-items: space-between;
+      grid-template-columns: repeat(4, 25%);
+
+  
+  }
+  
+  
+  .thumbnails{
+      width: 100%;
+      height: 150px;
+      object-fit: cover;
+      border-radius: 5px;
+  }
+  .channel-icon{
+      width: 40px;
+      height: 40px;
+      border: 2px solid black;
+      border-radius:50%;
+      object-fit: cover;
+      margin-right: 10px;
+  }
+  .information{
+      width: 100%;
+      height: 40px;
+      overflow: hidden;
+  }
+  .video-main{
+    position: fixed;
+    display: flex;
+    justify-content: center;
+    width: 100%;
+    margin-top: 30px;
+    margin-left: 260px;
+    padding: 2px ;
+  }
+  
+  .main-video{
+      position: fixed;
+      
+      width: 500px;
+      justify-content: center;
+      align-content: center;
+      margin-top: 30px;
+      margin-left: 260px;
+      padding: 2px;
+
+         
+  }
+  .main-icon{
+      display: flex;
+      justify-content: start;
+      width: 40px;
+      height: 40px;
+      margin-top: 350px;
+  
+      border: 2px solid black;
+      border-radius:50%;
+  
+  
+  } 
+  
```

Comment:
Please follow the CSS best practice to make your code perform correctly as required.

Reply:
Kindly explain what you mean by "best practices". What exactly practices should the student follow? The reason you provided for the requested change ("to make your code perform correctly as required") is too vague, not to mention it is obvious. Please provide an important and easy-to-understand reason for applying the requested practices.